gateway-ha/src/main/java/io/trino/gateway/ha/config/HaGatewayConfiguration.java

```diff
@@ -97,6 +98,16 @@ public RoutingConfiguration getRouting()
         return routing;
     }
 
+    public void setIncludeClusterHostInResponse(boolean includeClusterHostInResponse)
+    {
+        this.includeClusterHostInResponse = includeClusterHostInResponse;
```

Comment:
Config setters usually `return this` which allows for chaining

Reply:
I see that all the other setters are returning void. Just wanted to be consistent. Let me know if we still want to change it.